sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/nlp/nlp_search_space.py

```diff
@@ -42,6 +42,16 @@ class NlpSearchSpace(RestTranslatableMixin):
     :param weight_decay: value of weight decay when optimizer is sgd, adam, or adamw. This must be a float in
             the range [0, 1], defaults to None
     :type weight_decay: Optional[Union[float, SweepDistribution]]
+
+
+    .. admonition:: Example:
+
+    .. literalinclude:: ../samples/ml_samples_automl_nlp.py
```

Comment:
You'll need an indent here for the literalinclude block or it won't render correctly.